app/assets/stylesheets/course/announcements.css.scss

```diff
@@ -0,0 +1,24 @@
+// Place all the styles related to the course/announcements controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+.course-announcements {
+  &.index {
+    .announcement {
+      padding-left: 8px;
+
+      .timestamp {
+        color: #999;
```

Comment:
I'm not sure. Maybe in a variables.css.scss file? See how the Bootstrap gem does it.